File: JAVA/Dpr.java
//deklarasi kelas/class, huruf pertama harus kapital

import java.util.Set;

public class Dpr {
    private String id;
    private String name;
    private String bidang;
    private String partai;


    //konstruktor
    //versi yang ga ada param
    public Dpr()
    {
        this.id = "";
        this.name = "";
        this.bidang = "";
        this.partai = "";
    }

    //versi yang pake param, kayak nya untuk otomatis input nama dari user
    public Dpr(String id, String name, String bidang, String partai)
    {
        this.id = id;
        this.name = name;
        this.bidang = bidang;
        this.partai = partai;
    }

    public String getId ()
    {
        return this.id;
    }

    public void setId(String id)
    {
        this.id = id;
    }

    public String getName ()
    {
        return this.name;
    }

    public void setName(String name)
    {
        this.name = name;
    }

    public String getBidang ()
    {
        return this.bidang;
    }

    public void setBidang(String bidang)
    {
        this.bidang = bidang;
    }

    public String getPartai ()
    {
        return this.partai;
    }

    public void setPartai(String partai)
    {
        this.partai = partai;
    }

    //method publik/ bisa disebut behaviours

    public void eat()
    {
        System.out.println(this.name + "is eating!");
    }

    public void sleep()
    {
        System.out.println(this.name + "is sleeping!");
    }

    //java ga butuh destruktor kaya cpp
}

File: PYTHON/Dpr.py
class Dpr:
    #atribut privat
    __id = ""
    __name = ""
    __bidang = ""
    __partai = ""
    
    def __init__(self, id = "", name = "", bidang = "", partai = ""):
        self.__id = id
        self.__name = name
        self.__bidang = bidang
        self.__partai = partai
    
    def get_name(self):
        return self.__name
    
    def set_name(self, name):
        self.__name = name
        
    def get_id(self):
        return self.__id
    
    def set_id(self, id):
        self.__id = id
        
    def get_bidang(self):
        return self.__bidang
    
    def set_bidang(self, bidang):
        self.__bidang = bidang
        
    def get_partai(self):
        return self.__partai
    
    def set_partai(self, partai):
        self.__partai = partai
        
    #public method/bisa disebut behaviours
    def eat(self):
        print(self.__name, "is eating!")
        
    def sleep(self):
        print(self.__name, "is sleeping!")